src/tsconfig.vscode.json

```diff
@@ -32,7 +32,6 @@
 		"./sql/workbench/api/common/**/*.ts", // 294 errors
 		"./sql/workbench/browser/**/*.ts", // 3162 errors
 		"./sql/workbench/contrib/assessment/**/*.ts", // 3164 errors
```

Comment:
Do the other counts need to be updated?

Reply:
Probably, not that big of a deal though

Reply:
Every once in a while I update the numbers, but it takes a while to get all the numbers up to date so I don't think its necessary for every PR to update the numbers.